nnpdf_data/nnpdf_data/new_commondata/ATLAS_TTBAR_13TEV_LJ_DIF/filter.py

```diff
@@ -1,7 +1,59 @@
 import yaml
-from validphys.commondata_utils import symmetrize_errors as se
-from validphys.commondata_utils import percentage_to_absolute as pta
-from validphys.commondata_utils import covmat_to_artunc as cta
```

Comment:
I think the `covmat_to_artunc` was supposed to become a generic function to be used by all (since there are some datasets by @Radonirinaunimi that also use it).

Unless you already talked among yourselves and decided to take this one out as well.

Reply:
Yes, the commondata utils will have to include the `cormat_to_covmat`, otherwise all of the collider DY datasets will break. This PR, unfortunately however, does not introduce/address the utils yet (in the form we discussed at the CM) - which will still have to be done in a separate PR.

And in that respect, I absolutely don't think such function should be re-implemented in all of the filters which require it as it is done here.